app/models/user.rb

```diff
@@ -63,4 +64,10 @@ def self.from_omniauth(auth)
 
     return user
   end
+
+  #rebuild
```

Comment:
@babiracka the method has `rebuild` in its name – I don't think the comment is necessary in this case